docs/xls-65d-single-asset-vault/reference/transactions/vaultset.md

```diff
@@ -57,11 +57,11 @@ Besides errors that can occur for all transactions, {% code-page-name /%} transa
 
 | Error Code            | Description                           |
 | :-------------------- | :-------------------------------------|
-| `tecOBJECT_NOT_FOUND` | Occurs if the `Vault` object with the specified `VaultID` does not exist on the ledger. |
-| `tecNO_PERMISSION`    | Occurs if the account submitting the transaction is not the `Owner` of the vault. |
-| `temMALFORMED`        | Occurs if the `Data` field is larger than 256 bytes. |
-| `tecLIMIT_EXCEEDED`   | Occurs if the _new_ `AssetMaximum` value is **lower** than the vault's _current_ `AssetTotal`. |
-| `temDISABLED`         | Occurs if the Permissioned Domains amendment is disabled and a `DomainID` is provided. |
-| `tecINVALID_DOMAIN`   | Occurs if a transaction attempts to set `DomainID` on a public vault. |
+| `tecNO_ENTRY`         | The transaction attempted to modify a vault that does not exist. Check the `VaultID` field of the transaction. |
+| `tecOBJECT_NOT_FOUND` | The `PermissionedDomain` object with the provided `DomainID` does not exist. |
+| `tecNO_PERMISSION`    | Occurs if the account submitting the transaction is not the `Owner` of the vault, or is trying to set a `DomainID` for a public vault. |
+| `temMALFORMED`        | Occurs if the transaction was not validly formatted. For example, the `Data` field is larger than 256 bytes. |
+| `tecLIMIT_EXCEEDED`   | Occurs if the _new_ `AssetsMaximum` value is **lower** than the vault's _current_ `AssetsTotal`. |
+| `temDISABLED`         | Occurs if either the Permissioned Domains amendment is disabled and a `DomainID` is provided, or the Single Asset Vault amendment is not enabled.  |
```

Comment:
Nit, for consistency. Also suggested rewording the `temDISABLED` description for clarity.

```suggestion
| `tecNO_PERMISSION`    | The account submitting the transaction is not the `Owner` of the vault, or is trying to set a `DomainID` for a public vault. |
| `temMALFORMED`        | The transaction was not validly formatted. For example, the `Data` field is larger than 256 bytes. |
| `tecLIMIT_EXCEEDED`   | The _new_ `AssetsMaximum` value is **lower** than the vault's _current_ `AssetsTotal`. |
| `temDISABLED`         | Either the Single Asset Vault amendment is not enabled, or a `DomainID` is provided and the Permissioned Domains amendment is not enabled.  |
```